website/sidebars.json

```diff
@@ -16,6 +16,7 @@
       "understanding-radicle/glossary",
       "understanding-radicle/faq"
     ],
-    "Building on Radicle": ["building-on-radicle/documentation"]
+    "Building on Radicle": ["building-on-radicle/documentation"],
+    "Trouble Shooting": ["trouble-shooting"]
```

Comment:
```suggestion
    "Troubleshooting": ["trouble-shooting"]
```

Reply:
On a second thought: https://github.com/radicle-dev/radicle-docs/pull/21/files#r522813404.

Reply:
Fixed in: https://github.com/radicle-dev/radicle-docs/pull/21/commits/4f69e2a10e0edd7040fe7f3a72590f93a7a4c21e.